plasmapy/utils/calculator/__init__.py

```diff
@@ -8,12 +8,21 @@
 import shlex
 import subprocess
 
+_description = """
+Plasma calculator is a tool that opens a page in a web browser for
+interactive calculation of plasma parameters.
+
+This tool is currently in the prototype stage and is expected to change
+in the future.  Please raise an issue in PlasmaPy's GitHub repository to
+provide suggestions and feedback.
+"""

```

Comment:
This is what shows up when we do `plasma-calculator --help`.

Reply:
Do you want to provide a link directly to the plasmapy's issue creation?

https://github.com/PlasmaPy/PlasmaPy/issues/new?assignees=&labels=&template=Bug_report.md

or

https://github.com/PlasmaPy/PlasmaPy/issues/new

or

https://github.com/PlasmaPy/PlasmaPy/issues/new/choose